src/components/FocusVariantHeaderControls.tsx

```diff
@@ -1,23 +1,66 @@
 import React from 'react';
-import { Button, Dropdown } from 'react-bootstrap';
-import { ButtonGroup, DropdownButton } from 'react-bootstrap';
 import { Integration } from '../services/external-integrations/Integration';
 import { PangoLineageIntegration } from '../services/external-integrations/PangoLineageIntegration';
 import { OutbreakInfoIntegration } from '../services/external-integrations/OutbreakInfoIntegration';
 import { WikipediaIntegration } from '../services/external-integrations/WikipediaIntegration';
 import { CoVariantsIntegration } from '../services/external-integrations/CoVariantsIntegration';
 import { useState } from 'react';
-import { FaDownload } from 'react-icons/fa';
 import { UsherIntegration } from '../services/external-integrations/UsherIntegration';
 import { sequenceDataSource } from '../helpers/sequence-data-source';
 import { TaxoniumIntegration } from '../services/external-integrations/TaxoniumIntegration';
 import { getCsvLinkToContributors, getCsvLinkToDetails, getLinkToFasta } from '../data/api-lapis';
-import { ExternalLink } from './ExternalLink';
 import { useDeepCompareMemo } from '../helpers/deep-compare-hooks';
 import { useAsync } from 'react-async';
 import { OrderAndLimitConfig } from '../data/OrderAndLimitConfig';
 import { NextcladeIntegration } from '../services/external-integrations/NextcladeIntegration';
 import { LapisSelector } from '../data/LapisSelector';
+import { ExternalLink } from './ExternalLink';
+
+// mui stuff
+import Button1 from '@mui/material/Button';
+import MenuItem from '@mui/material/MenuItem';
+import { styled, alpha } from '@mui/material/styles';
+import Button from '@mui/material/Button';
+import Menu, { MenuProps } from '@mui/material/Menu';
+import DownloadIcon from '@mui/icons-material/Download';
+import Divider from '@mui/material/Divider';
+import KeyboardArrowDownIcon from '@mui/icons-material/KeyboardArrowDown';
+
+const StyledMenu = styled((props: MenuProps) => (
+  <Menu
+    elevation={0}
+    anchorOrigin={{
+      vertical: 'bottom',
+      horizontal: 'right',
+    }}
+    transformOrigin={{
+      vertical: 'top',
+      horizontal: 'right',
+    }}
+    {...props}
+  />
+))(({ theme }) => ({
+  '& .MuiPaper-root': {
+    'borderRadius': 6,
+    'marginTop': theme.spacing(1),
+    'minWidth': 180,
+    'color': theme.palette.mode === 'light' ? 'rgb(55, 65, 81)' : theme.palette.grey[300],
+    'boxShadow':
+      'rgb(255, 255, 255) 0px 0px 0px 0px, rgba(0, 0, 0, 0.05) 0px 0px 0px 1px, rgba(0, 0, 0, 0.1) 0px 10px 15px -3px, rgba(0, 0, 0, 0.05) 0px 4px 6px -2px',
+    '& .MuiMenu-list': {
+      padding: '4px 0',
+    },
+    '& .MuiMenuItem-root': {
+      '& .MuiSvgIcon-root': {
+        fontSize: 18,
+        color: theme.palette.text.secondary,
+      },
+      '&:active': {
+        backgroundColor: alpha(theme.palette.primary.main, theme.palette.action.selectedOpacity),
+      },
+    },
```

Comment:
I'm also a bit surprised to see this code. This does not look like the React-style. Are you sure that this is the right way to do it?